contents/security/passwordStorage.md

```diff
@@ -0,0 +1,106 @@
+<frontmatter>
+  title: Introduction to Password Storage
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Password Storage
+
+Author: [Jeremy Choo](https://github.com/ChooJeremy)
+
+## Overview
+
+While developing your own application, it is very common to implement login systems to be able to identify users. Whenever there is a need to store user-specific information, developers typically implement user accounts to identify these users separately. You might have seen ways this are done in other websites - using registration forms and requring you to login with a username and password to identify you for the session.
+
+As a developer, storing these usernames and passwords are essential to be able to refer to them later. While there are many ways to store these data - from text files to databases - this article will focus more on how to store passwords properly. For the purpose of this article, we will assume that you are storing the passwords in a database. Modern password mechanism typically involve **salting** and **hashing** the password. This article will describe what these mean and why they are important.
+
+### What's wrong with <trigger for="pop:plaintext">plaintext</trigger>?
+
+<popover id="pop:plaintext" title="_Plaintext_ refers to unencrypted information" placement="top">
+  <div slot="content">
+It is a cryptography term generally referring to text before encyption or after decrypting it. Another term for it is cleartext.
+  </div>
+</popover>
+
+While usernames are typically public information, thus <trigger for="pop:encrypt">encrypting</trigger> them is not as important, passwords are typically secret information and require encryption. If passwords are stored in plaintext and your server's database is ever compromised, attackers can easily use the stored plaintext passwords to login as the user and potentially access other functionality. This grants the attacker more power than just reading values off the database and being unable to read them if they were encrypted.
```

Comment:
I think you can rephrase it:

> Since usernames are typically public information, encrypting them is not as important. However, passwords are typically secret information and hence, require encryption.